set.mm

```diff
@@ -354553,6 +354589,103 @@ be obtained in Metamath (without the need for a new definition) by the curry
       DWDOAWHDWEWFWG $.
   $}
 
+  $( The general logarithm of a positive real number with respect to a real
+     base greater than 1 is positive iff the number is greater than 1.
+     (Contributed by AV, 29-Dec-2022.) $)
+  logbgt0b $p |- ( ( A e. RR+ /\ ( B e. RR+ /\ 1 < B ) )
+                   -> ( 0 < ( B logb A ) <-> 1 < A ) ) $=
```

Comment:
I don't know how your proof goes precisely, but if you proved the intermediate result
> |- ( 0 < A -> 0 <= (A-1) log A )

then it might be worth having it as a standalone.

Reply:
No I used ~loggt0b to show that `( 0 < ( log `` A ) <-> 1 < A )` and then I divided both sides of the greater than relation on the left hand side by `( log `` B ) )`, which is greater than 0 because of `1 < B`.